src/app/rundown-view/components/offsetable-part/offsetable-part.component.html

```diff
@@ -1,23 +1,18 @@
-<div [cdkContextMenuTriggerFor]="contextMenu" class="piece-layers"
-     [class.unsynced]="part.isUnsynced"
->
-    <div
-        *ngFor="let outputLayer of outputLayers"
-        class="piece-layer"
-    >
-        <sofie-offsetable-piece
-                *ngFor="let piece of piecesGroupedByOutputLayer[outputLayer]; trackBy: trackPiece"
-                [piece]="piece"
-                [pixelsPerSecond]="pixelsPerSecond"
-                [partDuration]="partDurationInMs()"
-                [playedDurationForPartInMs]="playedDurationInMs"
-                [prePlayheadDurationInMs]="prePlayheadDurationInMs"
-                [postPlayheadDurationInMs]="postPlayheadDurationInMs"
-        ></sofie-offsetable-piece>
-    </div>
+<div [cdkContextMenuTriggerFor]="contextMenu" class="piece-layers" [class.unsynced]="part.isUnsynced">
+  <div *ngFor="let outputLayer of outputLayers" class="piece-layer">
+    <sofie-offsetable-piece
+      *ngFor="let piece of piecesGroupedByOutputLayer[outputLayer]; trackBy: trackPiece"
+      [piece]="piece"
+      [pixelsPerSecond]="pixelsPerSecond"
+      [partDuration]="partDurationInMs()"
+      [playedDurationForPartInMs]="playedDurationInMs"
+      [prePlayheadDurationInMs]="prePlayheadDurationInMs"
+      [postPlayheadDurationInMs]="postPlayheadDurationInMs">
+    </sofie-offsetable-piece>
+  </div>
 </div>
-<sofie-timeline-flag [label]="part.isNext ? 'NEXT' : undefined" [turnedOn]="part.isNext"></sofie-timeline-flag>
+<sofie-timeline-flag [label]="part.isNext ? (isAutoNextStarted ? 'AUTO' : 'NEXT') : undefined" [turnedOn]="part.isNext"></sofie-timeline-flag>
```

Comment:
The `AUTO` and `NEXT` should ideally also be translated.

Also, just a note on nesting ternaries. As a rule of thumb we should try to avoid nesting ternaries as they have a tendency to become quite unreadable quite fast when they are nested.
The usage of it here is still fine, I think. I just wanted to give a headsup that we should be very mindful when we decide to nest ternaries :)

Reply:
Done